CHANGELOG.next.asciidoc

```diff
@@ -85,6 +85,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Journalbeat*
 
+- Add backoff again. {pull}11861[11861]
```

Comment:
Maybe make it more descriptive.

Journalbeat will now uses backoff when no events are found.